pkg/metrics/prometheus.go

```diff
@@ -4,11 +4,35 @@ import (
 	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"ktbs.dev/teler/common"
 )
 
 // PrometheusInsert logs into metrics
-func PrometheusInsert(data map[string]string) {
+func PrometheusInsert(options *common.Options, data map[string]string) {
 	var counter prometheus.Counter
+	cfg := options.Configs
+
+	//Check if teler have customs rules
+	if cfg.Rules.Threat.Customs != nil {
+		for _, customRule := range cfg.Rules.Threat.Customs {
+			if strings.HasPrefix(data["category"], customRule.Name) {
```

Comment:
What does this condition mean?

Reply:
I forget to change it into `//Check if user has customs rules` 

basically, it's to check if the user defines the custom rule or not

Reply:
Got it! Thus I believe that this can be reworked cleaner & less typing i.e.:
```go
	rules := make(map[string]bool)
	for _, customs := range cfg.Rules.Threat.Customs {
		rules[customs.Name] = true
	}

	switch {
	case rules[data["category"]]:
		// do stuff here
```

Reply:
yeah i think this was better